app/layout/header.tsx

```diff
@@ -11,6 +12,7 @@ export const Header = () => {
                         <div className="items-center">Alpha Dojo</div>
                         <div className="flex gap-2">
                             <Button variant="outline"><Play />Run</Button>
+                            <PrimaryButton title="Test2" onClick={()=>{alert("success")}}></PrimaryButton>
```

Comment:
nits: こちらも、=> の両側にスペースを付けたほうがいいですねー
あとは、関数呼出しを一行で行う場合は波括弧は無くても動きます。また、タグの中に要素を入れない場合は閉じタグは書かなくても ok ですｂ

```suggestion
<PrimaryButton title="Test2" onClick={() => alert("success")} />
```